### PR TITLE
Python3 compatibility decode byte string

### PR DESCRIPTION
**Pull request by miaxael**
fix for using Python3

### CODE REVIEW

**Comment by kshcherban** [comment]
Strange, i tested it with python3, can you provide an error you encountered?

**Comment by miaxael** [comment]
i get:
Nov 02 16:36:10 Writing result file in /etc/ssl/private/letsencrypt-domain.pem
Nov 02 16:36:10 Error writing cert: TypeError write() argument must be str, not bytes
Nov 02 16:36:10 removing /etc/nginx/sites-enabled/letsencrypt

then in  /etc/ssl/private/letsencrypt-domain.pem
the chain certificate is missing

File: acme_nginx/client.py [hunk 1]
```diff
@@ -324,6 +324,8 @@ def _log(message, exit_with_error=False):
         _log('Getting chain from {0}'.format(CHAIN))
         resp = urlopen(CHAIN)
         chain_str = resp.read()
+        if chain_str.decode:
```
Comment (kshcherban) [inline acme_nginx/client.py]: Probably should be instead
```
if chain_str:
```

File: acme_nginx/client.py [hunk 2]
```diff
@@ -324,6 +324,8 @@ def _log(message, exit_with_error=False):
...
+            chain_str = chain_str.decode('UTF-8')
```
Comment (kshcherban) [inline acme_nginx/client.py]: Please change to `utf8` instead as already used in code before.

**Pull request merged**